pkg/api/routers/metric.go

```diff
@@ -24,10 +25,11 @@ func init() {
 
 	// add router for metric api
 	metricns :=
-		beego.NewNamespace("/"+constants.APIVersion+":tenantId/metrics",
+		beego.NewNamespace("/"+constants.APIVersion+"/:tenantId/metrics",
 
 			// All operations of metrics can be used for both admin and users.
 			beego.NSRouter("/", controllers.NewMetricsPortal(), "get:GetMetrics"),
+			beego.NSRouter("/", controllers.NewMetricsPortal(), "post:CollectMetrics"),
```

Comment:
Please merge line 31 and 32 into a single one like below:
```go
beego.NSRouter("/", controllers.NewMetricsPortal(), "post:CollectMetrics,get:GetMetrics")
```

Reply:
merged